lib/SILOptimizer/Differentiation/JVPCloner.cpp

```diff
@@ -1006,11 +1073,13 @@ class JVPCloner::Implementation final
       return;
     }
     // Emit tangent `struct_element_addr`.
-    auto tanOperand = getTangentBuffer(bb, seai->getOperand());
+    auto tanOperand =
+        getTangentBuffer(bb, seai->getOperand(), InitializationRequired);
     auto tangentInst =
         diffBuilder.createStructElementAddr(loc, tanOperand, tanField);
     // Update tangent buffer map for `struct_element_addr`.
     setTangentBuffer(bb, seai, tangentInst);
+    tangentBufferInitializationInfo[tangentInst] = Initialized;
```

Comment:
`struct_element_addr` is valid on an uninitialized buffer. So I would probably let `tangentBufferInitializationInfo ]tangentInst]` inherit from `tangentBufferInitializationInfo[tanOperand]`.